src/lib/backup-copies/backup-copy-manager.ts

```diff
@@ -0,0 +1,615 @@
+import xdgBasedir from 'xdg-basedir';
```

Comment:
I'll rearrange the imports. Don't worry, I didn't forget.

It's just that I usually do this last because auto-import speeds things up so fast.